client/static/coderogue.css

```diff
@@ -34,6 +34,9 @@ span {
 .table tbody tr {
     height: 1.2rem;
 }
+#players tr:not(:nth-child(1)) {
```

Comment:
`tr:not(:nth-child(1))` is to skip the table header, which is not clickable. Wondering if there is better way to do this though.